pages/instructorFileBrowser/instructorFileBrowser.js

```diff
@@ -227,20 +228,18 @@ function browseFile(file_browser, callback) {
                 debug(`isBinaryFile: ${result}`);
                 file_browser.isBinary = result;
                 if (result) {
-                    try { // FIXME (check for PDF, etc.)
-                    const type = fileType(contents);
-                        debug(`file type: ${type}`);
-                        if (type) {
+                    util.callbackify(async () => {
+                        // FIXME (check for PDF, etc.)
+                        const type = await FileType.fromBuffer(contents);
+                        if (!type) {
```

Comment:
Just to check, is this really supposed to be `!type` rather than `type`?

Reply:
haha, yep

I was testing - thx